manifests/kube/kube.json

```diff
@@ -2,45 +2,45 @@
   "name": "kube",
   "description": "A plugin to manage spin apps",
   "homepage": "https://github.com/spinkube/spin-plugin-kube",
```

Comment:
```suggestion
  "homepage": "https://github.com/spinframework/spin-plugin-kube",
```

Reply:
there is a PR in spin-plugin-kube to update this in template: https://github.com/spinframework/spin-plugin-kube/pull/107